ds3-sdk-integration/src/test/java/com/spectralogic/ds3client/integration/UsersAndGroups_Test.java

```diff
@@ -0,0 +1,1229 @@
+                        putGlobalBucketAclForUserSpectraS3Response.getBucketAclResult()
+                                .getId().toString()));
+            }
+        }
+    }
+
+    @Test
+    public void putGlobalBucketAclForGroupString() throws IOException, SignatureException {
+        PutGlobalBucketAclForGroupSpectraS3Response putGlobalBucketAclForGroupSpectraS3Response = null;
+        try {
+            putGlobalBucketAclForGroupSpectraS3Response = client.putGlobalBucketAclForGroupSpectraS3(
+                    new PutGlobalBucketAclForGroupSpectraS3Request("Administrators",
+                            BucketAclPermission.DELETE));
+
+            assertThat(putGlobalBucketAclForGroupSpectraS3Response.getStatusCode(), is(201));
+
+        } finally {
+            if (putGlobalBucketAclForGroupSpectraS3Response != null) {
+                client.deleteBucketAclSpectraS3(new DeleteBucketAclSpectraS3Request(
+                        putGlobalBucketAclForGroupSpectraS3Response.getBucketAclResult()
+                                .getId().toString()));
+            }
+        }
+    }
+
+    @Test
+    public void putGlobalBucketAclForGroupUUID() throws IOException, SignatureException {
+        PutGlobalBucketAclForGroupSpectraS3Response putGlobalBucketAclForGroupSpectraS3Response = null;
+        try {
+            putGlobalBucketAclForGroupSpectraS3Response = client.putGlobalBucketAclForGroupSpectraS3(
+                    new PutGlobalBucketAclForGroupSpectraS3Request(administratorsUUID,
+                            BucketAclPermission.DELETE));
+
+            assertThat(putGlobalBucketAclForGroupSpectraS3Response.getStatusCode(), is(201));
+
+        } finally {
+            if (putGlobalBucketAclForGroupSpectraS3Response != null) {
+                client.deleteBucketAclSpectraS3(new DeleteBucketAclSpectraS3Request(
+                        putGlobalBucketAclForGroupSpectraS3Response.getBucketAclResult()
+                                .getId().toString()));
+            }
+        }
+    }
+
+    @Test
+    public void deleteBucketAcl() throws IOException, SignatureException {
+        PutBucketAclForUserSpectraS3Response putBucketAclForUserSpectraS3Response = null;
+        try {
+            putBucketAclForUserSpectraS3Response = client.putBucketAclForUserSpectraS3(
+                    new PutBucketAclForUserSpectraS3Request(BUCKET_NAME, BucketAclPermission.DELETE,
+                            spectraUUID));
+            final DeleteBucketAclSpectraS3Response deleteBucketAclSpectraS3Response = client
+                    .deleteBucketAclSpectraS3(new DeleteBucketAclSpectraS3Request(
+                            putBucketAclForUserSpectraS3Response.getBucketAclResult().getId().toString()));
+
+            assertThat(deleteBucketAclSpectraS3Response.getStatusCode(), is(204));
+
+        } catch (final Exception e){
+            if (putBucketAclForUserSpectraS3Response != null) {
```

Comment:
Is there a reason why a catch block is being used here instead of a finally block?